src/addspot.h

```diff
@@ -25,5 +25,6 @@
 
 void add_to_spots(char *call, freq_t freq);
 void addspot(void);
+void send_spot(void);
```

Comment:
maybe we can come up with more specific names.

Reply:
How about

'addspot' -> 'add_to_local_spots'
'send_spot' -> 'add_to_cluster_spots'

Reply:
Or just `add_local_spot`/`add_cluster_spot`? (or `send_`)

Reply:
Thats even better. Ok.